packages/stryker-elements/src/lib/atoms/dropdown.ts

```diff
@@ -0,0 +1,36 @@
+import { property } from 'lit/decorators.js';
+import { BaseElement } from '../base-element';
+import { html } from 'lit';
+import { map } from 'lit/directives/map.js';
+
+export class Dropdown extends BaseElement {
+  @property()
+  id: string = '';
+
+  @property()
+  options: Array<{ name: string; value: string }> = [];
```

Comment:
```suggestion
  options: { name: string; value: string }[] = [];
```

If this is valid syntax.